src/components/shared/__tests__/ChatListItem.test.tsx

```diff
@@ -69,8 +70,18 @@ describe('[ChatListItem] interaction', () => {
   });
 
   it('should fireEvent when peer image is pressed', () => {
-    const touchPeerImage = testingLib.getByTestId('peer_image');
+    // If I don't add testID to props (line 50), line 75~77 doesn't work.
+    // 1) Should I add testID to props (line 50)? Then, Why is there testID in ChatListIten.tsx?
+    const touchPeerImage = testingLib.getByTestId('chatListItem0');
     fireEvent.press(touchPeerImage);
     expect(cnt).toEqual(1);
+
+    // 2) Though I added testID to props(line 50), this codes(line 81~85) doesn't work.
```

Comment:
Actually, you are trying to to the same test which is already covered in previous paragraph.

To review your below test code, the error is appearing because of two reasons.
1. You should `mock`the function to test whether it's called using jest api `toHaveBeenCallledTimes`. Just to practice this, try to make `const onPressTest = jest.fn()`.
```
const onPressPeerImage = (): void => {
  cnt++;
  onPressTest();
};

// Then below will work (However, this is meaningless)
expect(onPressTest).toHaveBeenCalledTimes(1);
```

To summarize, you have nothing to do to add `coverage` in [ChatListItem] additionally.

Reply:
Actually, when I've checked `codecov` just now, you can increase [this line](https://codecov.io/gh/dooboolab/hackatalk-mobile/src/5c0e7a999cb2f73501df778b429e5048095aa251/src/components/shared/ChatListItem.tsx#L106). Try to pass `photoURL` with string and render it.

Answer is below (just adding below line in [ChatListItem.test.tsx]) will cover 100% of [ChatListItem.test.tsx].
```
  it('renders [peerMessage] with URL as expected', () => {
    props.item.sender.photoURL = 'https://';
    component = (
      <ThemeProvider theme={createTheme(ThemeType.LIGHT)}>
        <ChatListItem {...props} />
      </ThemeProvider>
    );
    const json = renderer.create(component).toJSON();
    expect(json).toMatchSnapshot();
  });
```

Reply:
Ok! I will add this code.